File: Exercicio_9.py
'''
Em uma fábrica de brinquedos, 5 máquinas produzem 60 peças em 4 horas. Durante o período de Natal,
a fábrica decidiu aumentar a produção para atender à demanda, colocando 8 máquinas para funcionar.
Quantas peças serão produzidas em 4 horas com esse aumento?

'''

#SOLUÇÃO 1, regra de tês:
def solucao_1():
    maquinas = 5
    maquinas_ex = 8
    qt_pecas = 60
    sub_total = maquinas_ex * qt_pecas
    total = sub_total / maquinas
    print('SOLUÇÃO_1, serão produzidas em 4 horas:',total)

#SOLUÇÃO 2:
def solucao_2():
    maquinas = 5
    maquinas_ex = 8
    qt_pecas = 60
    subtotal = qt_pecas / maquinas
    total = subtotal*maquinas_ex
    print('SOLUÇÃO_2, serão produzidas em 4 horas:', total)

solucao_1()
solucao_2()

File: Exercicio_9.java
package exercicio_9;

/*Em uma fábrica de brinquedos, 5 máquinas produzem 60 peças em 4 horas. Durante o período de Natal,
a fábrica decidiu aumentar a produção para atender à demanda, colocando 8 máquinas para funcionar.
Quantas peças serão produzidas em 4 horas com esse aumento?*/

public class Exercicio_9 {

	public static void main(String[] args) {
		// Exibi quantas peças são produzidas em 4 horas
		System.out.println("Foram produzidas: " + solucao_1() + " peças.");
		System.out.println("Foram produzidas: " + solucao_2() + " peças.");

	}

	//solução regras de três
	public static int solucao_1() {

		int maquinas = 5;
		int maquinas_ex = 8;
		int qt_pecas = 60;
		int sub_total = maquinas_ex * qt_pecas;

		return sub_total / maquinas;
	}

	//solução apresentada pelo autor
	public static int solucao_2() {
		
		int maquinas = 5;
	    int maquinas_ex = 8;
        int qt_pecas = 60;
	    int subtotal = qt_pecas / maquinas;
		return subtotal*maquinas_ex;
	}

}
